java/integTest/src/test/java/glide/modules/VectorSearchTests.java

```diff
@@ -220,4 +230,376 @@ public void ft_drop() {
         assertInstanceOf(RequestException.class, exception.getCause());
         assertTrue(exception.getMessage().contains("Index does not exist"));
     }
+
+    @SneakyThrows
+    @Test
+    public void ft_aggregate() {
+        var prefixBicycles = "{bicycles}:";
+        var indexBicycles = prefixBicycles + UUID.randomUUID();
+        var prefixMovies = "{movies}:";
+        var indexMovies = prefixMovies + UUID.randomUUID();
+
+        // FT.CREATE idx:bicycle ON JSON PREFIX 1 bicycle: SCHEMA $.model AS model TEXT $.description AS
+        // description TEXT $.price AS price NUMERIC $.condition AS condition TAG SEPARATOR ,
+        assertEquals(
+                OK,
+                FT.create(
+                                client,
+                                indexBicycles,
+                                new FieldInfo[] {
+                                    new FieldInfo("$.model", "model", new TextField()),
+                                    new FieldInfo("$.description", "description", new TextField()),
+                                    new FieldInfo("$.price", "price", new NumericField()),
+                                    new FieldInfo("$.condition", "condition", new TagField(',')),
+                                },
+                                FTCreateOptions.builder()
+                                        .indexType(IndexType.JSON)
+                                        .prefixes(new String[] {prefixBicycles})
+                                        .build())
+                        .get());
+
+        // TODO use JSON module API
+        // TODO check JSON module loaded
+
+        client
+                .customCommand(
+                        new String[] {
+                            "JSON.SET",
+                            prefixBicycles + 0,
+                            ".",
+                            "{\"brand\": \"Velorim\", \"model\": \"Jigger\", \"price\": 270, \"description\":"
```

Comment:
are you getting the bicycle data from somewhere?  we should add a link to the reference if so.

Reply:
Adopted from https://github.com/RedisVentures/redis-vss-getting-started/blob/main/data/bikes.json